pkg/api/login_handler.go

```diff
@@ -21,11 +21,16 @@ const (
 	JWTCookieName          = "access_token"
 )
 
-func GenerateJWT(secret []byte, userID int, issuedAt, expiresAt time.Time) (string, error) {
+// Be aware! The aud field should *only* be used for new applications! Otherwise it should be passed as an empty string i.e. pass LoginAudience or the proper audience,
```

Comment:
godoc comments should begin "GenerateJWT" and then a present-tense description of what it does.  I believe this has already been mentioned; did you forget to push?

Reply:
Fixed the docs according to the new code. Tried my best to write it by the godoc rules.